jni/src/nmslib_wrapper.cpp

```diff
@@ -195,7 +195,7 @@ jobjectArray knn_jni::nmslib_wrapper::QueryIndex(knn_jni::JNIUtilInterface * jni
     int resultSize = neighbors->Size();
 
     jclass resultClass = jniUtil->FindClass(env,"org/opensearch/knn/index/KNNQueryResult");
-    jmethodID allArgs = jniUtil->FindMethod(env, resultClass, "<init>", "(IF)V");
+    jmethodID allArgs = jniUtil->FindMethod(env, "org/opensearch/knn/index/KNNQueryResult", "<init>");
```

Comment:
Is this intended? We seem to still keep the class name?

Reply:
Yes, FindMethod is a wrapper defined in JNIUtil. We pass it the class name and the method name and it finds the method in the map.